forms-flow-web/src/components/Form/List.js

```diff
@@ -749,6 +750,10 @@ const mapDispatchToProps = (dispatch, ownProps) => {
       if (!applicationCount) {
         dispatch(deleteForm("form", formId));
       }
+      // delete drafts when form delete
+      deleteDraftbyFormId(formId).catch(err=>{
```

Comment:
Cant we trigger this from backend itself . Add code to call draft delete function when form delete is done @sreehari-aot / @shuhaib-aot

Reply:
better in that way @abhilash-aot

Reply:
ok